web/packages/core/src/internal/player/impl_v1.ts

```diff
@@ -0,0 +1,109 @@
+import { PlayerV1 } from "../../public/v1";
+import { InnerPlayer, ReadyState } from "./inner";
+import type { DataLoadOptions, URLLoadOptions } from "../../load-options";
+import type { MovieMetadata } from "../../movie-metadata";
+
+export class PlayerV1Impl implements PlayerV1 {
+    #inner: InnerPlayer;
+
+    constructor(inner: InnerPlayer) {
+        this.#inner = inner;
+    }
+
+    get onFSCommand(): ((command: string, args: string) => boolean) | null {
```

Comment:
3 possibilities here:
1. As today, we set **the** callback for **all** FS commands.
2. We change it to register **a** callback for **all** FS commands.
3. We change it to register **the** callback for **a specific named** FS command.

I think I kinda prefer 2.